7.10/alpine/Dockerfile

```diff
@@ -0,0 +1,68 @@
+FROM alpine:3.4
```

Comment:
We dropped Node.js 7.10 awhile ago. I think your branch is out of sync with master.